privacy-policy.md

```diff
@@ -0,0 +1,183 @@
+Please read this privacy policy carefully as it will help you make informed decisions about sharing your personal information with us.
+
+Table of contents:  
+    1. WHAT INFORMATION DO WE STORE?  
+    2. HOW DO WE USE YOUR INFORMATION?  
+    3. WILL YOUR INFORMATION BE SHARED WITH ANYONE?  
+    4. DO WE USE COOKIES AND OTHER TRACKING TECHNOLOGIES?  
+    5. HOW DO WE HANDLE YOUR SOCIAL LOGINS?  
+    6. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?  
+    7. WHAT IS OUR STANCE ON THIRD-PARTY WEBSITES?  
+    8. HOW LONG DO WE KEEP YOUR INFORMATION?  
+    9. HOW DO WE KEEP YOUR INFORMATION SAFE?  
+    10. DO WE COLLECT INFORMATION FROM MINORS?  
+    11. WHAT ARE YOUR PRIVACY RIGHTS?  
+    12. DO CALIFORNIA RESIDENTS HAVE SPECIFIC PRIVACY RIGHTS?  
+    13. DO WE MAKE UPDATES TO THIS POLICY?  
+    14. HOW CAN YOU CONTACT US ABOUT THIS POLICY?  
+
+## WHAT INFORMATION DO WE STORE? 
+
+### **Personal information you disclose to us**
+
+_In Short: We store personal information that you provide to us such as email-address._  
+
+We store personal information that you voluntarily provide to us when registering at the Site, 
+and further on by editing your profile at the Site.
+The personal information that we store depends on the context of your interactions 
+with us and the Site, the choices you make and the products and features you use. The personal information we store can include the following: 
+
+- We store username, email address and social media usernames. Username and email address are mandatory.
+- Credentials. We store passwords. All of the Credentials are hashed and original values not stored.
+- All personal information that you provide to us must be true, complete and accurate, and you must notify us of any changes to such personal information.
+
+### **Information automatically stored**  
+
+_In Short: Certain information – such as IP address and/or browser and device characteristics – is stored automatically when you visit our websites._  
+
+We automatically store certain information when you visit, use or navigate the Site. This information does not reveal your specific identity (like your name or contact information) but may include device and usage information, such as your IP address, browser and device characteristics, operating system, information about how and when you use our Site and other technical information. This information is primarily needed to maintain the security and operation of our Site, and for our internal analytic and reporting purposes.  
+
+
+We'd like to state that there are 2 different automatic storing mechanism:
+
+The first one is, server logs which has this certain infromation stored and are flushed every 24 hours.
+This is primarily needed for security and maintaining the operation of our Site.
+
+The second one is targetted for analytics, this mechanism on happen on certain portions of the site. This mechanism, only stores an hashed version of your IP.
```

Comment:
Hmm, what happens if we do this on a new route? I don't want to make it an habit by updating the policy once in a while.

Thereby it's hard speaking of technical terms so instead of routes call them "By visiting one of these URL path" as Path is the more broad defined name for `/hello/this/is/an/path` within legal terms. If we want to define it with good legal terms, within the IETFT RFC 1738 https://datatracker.ietf.org/doc/html/rfc1738 it's defined as relative link. But route? Thats some made up name by us as developers.

Reply:
I doubt we'll have more routes than that, but we could leave these two purely as examples.

Up to you how we word it, but it's worth noting that we only do this where it makes sense to provide necessary functionality, and not everywhere like all other services that collect analytics.